3ds Max/Max2Babylon/Forms/ExporterForm.cs

```diff
@@ -187,12 +204,14 @@ private void SaveOptions()
             Loader.Core.RootNode.SetStringProperty(ExportParameters.ModelFilePathProperty, Tools.RelativePathStore(unformattedPath));
 
             string unformattedTextureFolderPath = PathUtilities.UnformatPath(txtTextureName.Text);
-            Loader.Core.RootNode.SetStringProperty(ExportParameters.TextureFolderPathProperty,Tools.RelativePathStore(unformattedTextureFolderPath));
+            Loader.Core.RootNode.SetStringProperty(ExportParameters.TextureFolderPathProperty, Tools.RelativePathStore(unformattedTextureFolderPath));
 
             Tools.UpdateCheckBox(chkFullPBR, Loader.Core.RootNode, ExportParameters.PBRFullPropertyName);
             Tools.UpdateCheckBox(chkNoAutoLight, Loader.Core.RootNode, ExportParameters.PBRNoLightPropertyName);
             string unformattedEnvironmentPath = PathUtilities.UnformatPath(txtEnvironmentName.Text);
             Loader.Core.RootNode.SetStringProperty(ExportParameters.PBREnvironmentPathPropertyName, Tools.RelativePathStore(unformattedEnvironmentPath));
+
+            Tools.UpdateCheckBox(chkFlatten, Loader.Core.RootNode, "babylonjs_flatten");
```

Comment:
can we use "babylonjs_flattenScene" as our flag name?

Reply:
done